src/main/scala/tile/FPU.scala

```diff
@@ -45,6 +45,7 @@ trait HasFPUCtrlSigs {
   val div = Bool()
   val sqrt = Bool()
   val wflags = Bool()
+  val vec = Bool()
```

Comment:
I hate the Rocket way of adding new decode fields...

Reply:
I agree... but this should never need to be changed again...